### PR TITLE
Add some of the missing BASH remediations

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Add some of the missing BASH remediations

#### Rationale:

- Scripts initially provided in #2494. Closing some of the missing BASH remediation gap.

### CODE REVIEW

File: rhel7/fixes/bash/disable_prelink.sh [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-#
-# Undo previous prelink changes to binaries
-#
-/usr/sbin/prelink -ua
+if rpm --quiet -q prelink; then
+	if grep -q ^PRELINKING /etc/sysconfig/prelink
+	then
+		sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
+	else
+		echo -e "\n# Set PRELINKING=no per security requirements" >> /etc/sysconfig/prelink
```
Comment (matejak) [inline rhel7/fixes/bash/disable_prelink.sh]: Shellcheck will complain about `\n` inside double quotes, so it is better to change those to single quotes.
https://github.com/koalaman/shellcheck/wiki/SC1117

File: shared/fixes/bash/no_rsh_trust_files.sh [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # platform = multi_platform_rhel
 find /home -maxdepth 2 -type f -name .rhosts -exec rm -f '{}' \;
-rm -f /etc/hosts.equiv
+
+if [ -f /etc/hosts.equiv ]; then
```
[the remark this answers is not in the document]
Comment (redhatrises) [reply shared/fixes/bash/no_rsh_trust_files.sh]: @matejak it's more to bring in line with the direction that we have been going with our bash scripts which is if the file is already configured or doesn't exist then don't run anything.

**Comment by matejak** [comment]
I am satisfied with this PR. Do you need somebody else to take look at it? If not, I will merge it.
On a unrelated note, I would be grateful for no significant rebases during PR creation - they break the PR evolution history. Anybody who gets an idea to learn how to contribute to SSG by example from old PRs will be disappointed, because there will be no process to learn from, there will be just that one commit that is 100% correct.

**Comment by redhatrises** [comment]
@matejak go ahead and merge it.

**Comment by matejak** [comment]
Great, awesome contribution!

**Pull request merged**